docs/openrpc.json

```diff
@@ -651,8 +657,9 @@
       "description": "![](https://raw.githubusercontent.com/hashgraph/hedera-json-rpc-relay/main/docs/images/http_label.png) ![](https://raw.githubusercontent.com/hashgraph/hedera-json-rpc-relay/main/docs/images/ws_label.png)",
       "params": [],
       "result": {
-        "name": "Always returns '0x0'. Hedera doesn't have a concept of tipping nodes to promote any behavior",
+        "name": "eth_maxPriorityFeePerGas result",
```

Comment:
QQ: I see the name has been changed around. Is this because we want the name to be explicit or because we have to align with the specification?

Reply:
- decided to change it to be consistent with the ethereum one and ours more or less;
- visually it's too long;
- in playground it makes more sense when you open the result to have additional info about it together with the type and pattern;
- and in my opinion this sentence is a some kind of a description or additional info about the result, not the title.

Reply:
nice thanks for confirming!